plugins/inventory/digitalocean.py

```diff
@@ -0,0 +1,186 @@
+# -*- coding: utf-8 -*-
+
+# Copyright: (c), Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+name: digitalocean
+plugin_type: inventory
```

Comment:
```suggestion
```
(https://github.com/ansible/ansible/issues/71795)

Reply:
I thought that it's a must have

Reply:
How can I run such a test? Is there a framework for this purpose?

Reply:
RTFM helped me out